sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -415,6 +415,21 @@ public void testGrantOnTablePrivilegeStmtBuilder() {
         printStatement("grant ALL ON TABLE my_schema.t, banana.b, tree to anna, nyan, cat");
     }
 
+
+    @Test
+    public void testCreateIngestRuleStmtBuilder() {
+        printStatement("CREATE INGEST RULE v4 ON mqtt WHERE topic like 'v4/%' INTO my_schema.raw_v4");
+        printStatement("CREATE INGEST RULE \"http_r1\" ON \"http\" WHERE topic like 'v4/%' INTO my_schema.raw_v4");
```

Comment:
afaik `http_r1` can be used without quoting, we should use some example here where the quoting is important like e.g. uppercase idents.